swiftgen.yml

```diff
@@ -10,5 +10,4 @@ xcassets:
     - CriticalMass/Images.xcassets
     outputs:
       - templateName: swift4
-        output: CriticalMass/Generated/Images.swift
-        
+        output: CriticalMass/Generated/Images.swift
```

Comment:
was this change intended?

Reply:
I do not get that change tbh. I think I reverted your commit by it so could you explain that one again? 4726ef0262ec0d7581f26b2adc90d0477e4c6a15

Reply:
Sure! I've added the newline because according to POSIX, every text file should end with a \n, or “newline” character.

You can read more about it here: https://stackoverflow.com/a/729795 

Maybe it's just nitpick...

In GitHub (only in the browser) the missing newline is marked with a red icon.

Reply:
Interesting. I'll revert that